mindone/trainers/ema.py

```diff
@@ -47,16 +46,25 @@ def ema_update(self):
         self.updates = F.depend(self.updates, success)
         return self.updates
 
-    # @ms_function
+    def swap_data(self, ori_datas, tgt_datas):
```

Comment:
so `set_data` is done in CPU and `assign` is done in NPU? it is a bit tricky, maybe add a explanation in the code as it is a public module